### PR TITLE
👷‍♀️ FIX: deprecate dangerously action

### PR DESCRIPTION
**Pull request by kenjinp**

### CODE REVIEW

**Comment by kenjinp** [comment]
This actions are built using the props and the keys of that object. it won't know to use the proper method unless you declare it as an argument to `makeActions`

File: src/shared/components/Resources/ResourceActions.tsx [hunk 1]
```diff
@@ -121,6 +121,7 @@ export interface ResourceActionsProps {
   goToElasticSearchView: (resource: Resource) => void;
   deprecateResource: (resource: Resource) => void;
   downloadFile: (resource: Resource) => void;
+  veryDangerousToDeprecate: (resource: Resource) => void;
```
Comment (julienmachon) [inline src/shared/components/Resources/ResourceActions.tsx]: What's the difference between `deprecateResource` and `veryDangerousToDeprecate` ?

Comment (kenjinp) [reply src/shared/components/Resources/ResourceActions.tsx]: one is the normal message, "are you sure?", and the other is "this will break stuff!"

Comment (julienmachon) [reply src/shared/components/Resources/ResourceActions.tsx]: I mean... we're not going to add a new prop everytime we need a different message right? can we not do something like:
```suggestion
  deprecateResource: (resource: Resource, message?: string) => void;
```

Comment (kenjinp) [reply src/shared/components/Resources/ResourceActions.tsx]: You're right, I fixed the actions to be a little more generic, now you can create a button that uses another action and you dont have to add a new prop type every time

**Comment by julienmachon** [comment]
Let's merge this for now with the idea in mind that we will refactor soon: https://github.com/BlueBrain/nexus/issues/688

**Pull request merged**